utils/1click_image_scripts/template-docker-compose.yml

```diff
@@ -41,6 +41,8 @@ services:
     restart: always
     ports:
       - '127.0.0.1:8000:80'
+    environment:
+      FILE_SIZE_LIMIT_MB: 100
```

Comment:
again, do we need it set in both containers?

Reply:
yes, because its also set on the nginx config

Reply:
see here
[    client_max_body_size ${FILE_SIZE_LIMIT_MB}m;](https://github.com/specklesystems/speckle-server/blob/10d02da741255c43641292201fe0cbbf3e1555b0/packages/frontend/nginx/templates/nginx.conf.template#L113)